mp/src/game/client/momentum/ui/HUD/hud_comparisons.cpp

```diff
@@ -360,7 +360,7 @@ void C_RunComparisons::GetDiffColor(float diff, Color *into, bool positiveIsGain
 // only obtaining the actual, only obtaining the comparison, or only obtaining the color.
 void C_RunComparisons::GetComparisonString(ComparisonString_t type, CMomRunStats *stats, int zone,
                                            char *ansiActualBufferOut, char *ansiCompareBufferOut,
-                                           Color *compareColorOut)
+                                           Color *compareColorOut, bool roundVelocity)
```

Comment:
First oldest panel the Timer, second oldest is probably the RunComparisons panel...

Nothing wrong here, other than the base code here being full of dust and awaiting the stats refactor coming in 0.10.0.

Reply:
Yeah it's pretty rough. Just felt this was a better solution than dealing with the unrounded string output then ripping the string apart to round it.